app/controllers/admin/blog_posts_controller.rb

```diff
@@ -43,8 +43,8 @@ def destroy
   end
 
   def get_tags
-    @tags = BlogPost.find_tags(tag_params)
-    render json: @tags.map(&:name)
+    tags = BlogPost.find_tags(tag_params)
+    render json: tags.collect { |tag| {value: tag.name} }.to_json
```

Comment:
Autocomplete default expects a list of hashes. 
Something like:
```
[{value: "tag1"}, {value: "tag2"}]
```
without adding to_json at the end rails returns:
```
{"blog_posts": [{value: "tag1"}, {value: "tag2"}]}
```
which isn't what autocomplete expects and so fails.